test.py

```diff
@@ -0,0 +1,162 @@
+# pyGene Fusion local testing
+import json
+from pydantic import BaseModel, validator, ValidationError, StrictStr, StrictInt, StrictBool
+from typing import Optional, List, Union
+
+class Gene(BaseModel):
+    type = 'GeneDescriptor'
+    value_id: str
+    label: StrictStr
+
+    @validator('value_id')
+    def is_valid(cls,v):
+        assert v.find(':') != -1 and v.find(' ') == -1, 'id must be a CURIE'
```

Comment:
I believe there should be exactly 1 colon in a string. 
```suggestion
        assert v.count(':') == 1 and v.find(' ') == -1, 'id must be a CURIE'
```

Reply:
Maybe since you use this assertion multiple times, you could create a CURIE class

Reply:
Good point on using count. I wasn't sure if I needed to create a separate class for CURIE but it's probably good to have considering we have CURIEs multiple times.

Reply:
Good point on using count. I wasn't sure if I needed to create a separate class for CURIE but it's probably good to have considering we have CURIEs multiple times.